receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -779,3 +779,21 @@ metrics:
       value_type: int
     attributes: [operation, index_aggregation_type]
     enabled: true
+  elasticsearch.index.cache.evictions:
+    description: The number of evictions from the cache for an index.
+    unit: "{evictions}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name, index_aggregation_type]
+    enabled: true
```

Comment:
We need to start restricting the number of metrics that are enabled by default. Basically, we need to ensure that only metrics that are particularly useful will be enabled by default. Unless you are sure this is the case, please mark this as false. The same rule tentatively will apply to all new metrics.